README.md

```diff
@@ -65,88 +65,88 @@ mOcKiNgCaSe('ABCDEF', {lower: 'bcd'});
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 
 
-- [mOcKiNgCaSe(input, [options]) ⇒ <code>string</code>](#mOcKiNgCaSe)
-- [mOcKiNgCaSe.overrideString() ⇒ <code>mOcKiNgCaSe</code>](#mOcKiNgCaSe.overrideString)
+- [mockingcase(input, [options]) ⇒ <code>string</code>](#mockingcase)
+- [mockingcase.overrideString() ⇒ <code>mockingcase</code>](#mockingcase.overrideString)
 - [String.prototype.toMockingCase([options]) ⇒ <code>string</code>](#String.prototype.toMockingCase)
-- [mOcKiNgCaSe.config(defaultOptions) ⇒ <code>mOcKiNgCaSe</code>](#mOcKiNgCaSe.config)
-- [mOcKiNgCaSe.log(input, [options])](#mOcKiNgCaSe.log)
-- [mOcKiNgCaSe.overrideConsole([options]) ⇒ <code>mOcKiNgCaSe</code>](#mOcKiNgCaSe.overrideConsole)
+- [mockingcase.config(defaultOptions) ⇒ <code>mockingcase</code>](#mockingcase.config)
+- [mockingcase.log(input, [options])](#mockingcase.log)
+- [mockingcase.overrideConsole([options]) ⇒ <code>mockingcase</code>](#mockingcase.overrideConsole)
 - [Browser Usage](#browser-usage)
 
-<a name="mOcKiNgCaSe"></a>
+<a name="mockingcase"></a>
 
-### mOcKiNgCaSe(input, [options]) ⇒ <code>string</code>
-Converts the input string(s) to mOcKiNgCaSe.
+### mockingcase(input, [options]) ⇒ <code>string</code>
+Converts the input string(s) to mockingcase.
```

Comment:
This is an explanation of that function do so here it should remain **Converts the input string(s) to mOcKiNgCaSe.** Please also check other lines like 162, 167 and others.